PROTOCOL.md

```diff
@@ -1343,6 +1344,17 @@ When enabled:
  - The value of `delta.generationExpression` SHOULD be parsed as a SQL expression.
  - Writers MUST enforce that any data writing to the table satisfy the condition `(<value> <=> <generation expression>) IS TRUE`. `<=>` is the NULL-safe equal operator which performs an equality comparison like the `=` operator but returns `TRUE` rather than NULL if both operands are `NULL`
 
+## Default Columns
+
+Delta supports defining default expressions for columns on Delta tables. Delta will generate default values for columns when users do not explicitly provide values for them when writing to such tables. To enable Default Columns:
+- The table must be on Writer Version 4, or
```

Comment:
GitHub UI is showing 4-line blocks. My comments are regarding the last line in each block.

Reply:
Good point, since this feature requires table features [1] it appears that writer version >= 7 is required. I removed this line accordingly.

[1] https://delta.io/blog/2023-07-27-delta-lake-table-features/